### PR TITLE
Configurable bandwidth estimation side

### PR DESCRIPTION
**Pull request by boks1971**
While the TWCC work is happening in Pion, wanted to set up server to be able to consume it when it is ready. As part of that, making which end the bandwidth estimation runs a config and choosing RTP header extensions/RTCP feedback properly. Also, moving those to config so that it is all in one place.

For now, commenting out code with LK-TODO-SSBWE marker to indicate portions that need to be enabled when consuming send side bandwidth estimation work.

### CODE REVIEW

File: pkg/rtc/config.go [hunk 1]
```diff
@@ -112,6 +131,41 @@ func NewWebRTCConfig(conf *config.Config, externalIP string) (*WebRTCConfig, err
...
+				sdp.TransportCCURI,
+				frameMarking,
+			},
+		},
+	}
+
+	// subscriber configuration
+	subscriberConfig := DirectionConfig{
+		RTCPFeedback: RTCPFeedbackConfig{
+			Video: []webrtc.RTCPFeedback{
```
Comment (davidzhao) [inline pkg/rtc/config.go:154]: I believe we need the RTCP video feedback for the publisher side as well?

**Review by davidzhao — APPROVED** [review]
lg, just one question.

File: pkg/rtc/config.go (hunk 1 above)
Comment (boks1971) [reply pkg/rtc/config.go:154]: Yeah, missed that. Will add it. Got thrown off by that code setting REMB. Will see publisher side to `transport-cc`.

Comment (boks1971) [reply pkg/rtc/config.go:163]: Seems to be not needed as Pion is replying with what is in the offer. But, does not hurt adding it. Also, adding both REMB and TWCC as both are supported in Pion.

**Pull request merged**